### DISCUSSION TITLE
Welcome to gpt_academic Discussions! QQ group 610599535

### DISCUSSION THREAD

**Discussion by binary-husky** [binary-husky/gpt_academic#247] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Misery7100** [comment]
Hello community!

I am Lev -  a student, researcher and frequent user of ChatGPT for solving routine tasks. You've done a solid perfect job! I'm wondering is some translation configuration and uniform code planned or not? Because I'd be very glad to collaborate and add features, but sometimes I spend more time with translator than on feature incorporation 😄 

Would love you feedback, thanks!

**Comment by WSL2022** [comment]
= =
![image](https://user-images.githubusercontent.com/99741353/229801294-95086557-53ef-4318-946f-ce302c9aea02.png)

**Reply by qiuxiaomu** [reply]
对于不清楚的同学, gpt-academic是调用openai的API执行问答的,不是在背后开了个小窗口然后悄悄输入了免费版chatGPT-3.5再反馈给我们.如果我们不是付费用户(也就是绑定了信用卡的用户), 你的账户是没有办法使用openai的API的, 这也就是我们为什么要提供API key的意义, gpt-academic会让openai验证key然后对key绑定的账户进行扣款; 你不付费, 那是不会让你白嫖的.

**Comment by 1ang1** [comment]
我运行了python -m pip install -r requirements.txt这段指令之后终端没有响应，可以请教一下有什么解决办法嘛？

**Reply by Masterlovsky** [reply]
> 我运行了python -m pip install -r requirements.txt这段指令之后终端没有响应，可以请教一下有什么解决办法嘛？

检查一下你的机器能不能连网，如果可以正常连网可能是国内连pip源连不上卡住了，在命令后面加上
"-i https://pypi.tuna.tsinghua.edu.cn/simple" 试试

**Comment by Masterlovsky** [comment]
能否将是否启用huggingface分享放到config.py中，不然每次都要改main.py好麻烦

**Comment by haoboy007** [comment]
我的是美国VPSLinux服务器，按docker安装方法做了，也填写了API，怎么按照http://服务器IP::58561，不能访问啊，求助！QQ：2411977441

**Comment by FufangFufang** [comment]
File "D:\chatgpt_academic\main.py", line 2, in <module>
    import gradio as gr
ModuleNotFoundError: No module named 'gradio'    为啥会有这样的问题啊？我什么都还没有动呢

**Comment by Yolooloy7** [comment]
运行main.py出现了这个ModuleNotFoundError: No module named 'resource'，有人能帮帮我么？Help!!!

**Comment by oyasumiaiko** [comment]
qq群为什么莫名其妙踢人? @binary-husky ? 从没发过广告, 聊的也都是LLM AI, 没任何说明没任何解释直接就被踢了, 搞什么?
